pages/users.tsx

```diff
@@ -69,10 +69,15 @@ const UsersPage = ({users}: { users: Array<User> }): React.ReactElement => {
   }, {})
 
   const handleSubmit = async (values: EditableUserValues, {setStatus}) => {
+    const changedUsers = Object.fromEntries(
+      Object.entries(values)
+        .filter(([index, value]) => JSON.stringify(value) !== JSON.stringify(initialValues[index]))
+    );
+
```

Comment:
sorry for being naive, why are we using filter instead of select here? like we select and return only the matched values.

Reply:
js doesn't have a `select` array method. there is a `find` though which works the same as filter but will only return a single resuilt

Reply:
thanks